routes/reminders.js

```diff
@@ -11,8 +11,22 @@ function removeFamilyProperty(item) {
 }
 
 router.get('/', (req, res, next) => {
-  reminders.index(req.user.family, req.query.start, req.query.limit)
-  .then(rows => {
+  const start = parseInt(req.query.start);
+  let limit = parseInt(req.query.limit);
```

Comment:
now I use parseInt because it gives "NaN" for text without numbers and a number when the string starts with a number. It makes it easier to reason about.